mycroft/skills/intent_service.py

```diff
@@ -17,13 +17,15 @@
 import time
 
 from mycroft.configuration import Configuration, set_default_lf_lang
+from mycroft.util.lang import set_active_lang
```

Comment:
This is actually used in reset converse() so if I remove it it might(?) break something. That pre-dates me and I have no idea about that code. Am I to understand that the feature/mark-2 branch is broken? I branched off feature/mark-2 and I did not change this code?

Reply:
Oh I see . this is because this PR is like 2 weeks old now and the underlying code has changed recently. OK, so what is it supposed to be. Is this what replaces set_active_lang?

set_default_lf_lang(lang)

Reply:
OK, I believe I have taken care of this